specs/ServerCertificate.md

```diff
@@ -0,0 +1,299 @@
+  string message = (isSuccessful) ?
+                        "Clear server certificate error override cache succeeded." :
+                        "Clear server certificate error override cache failed.";
+  MessageBox.Show(this, "message", "Clear server certificate error override cache");
+}
+```
+
+# API Details
+
+## Win32 C++
+``` cpp
+[uuid(4B7FF0D2-8203-48B0-ACBF-ED9CFF82567A), object, pointer_default(unique)]
+interface ICoreWebView2_11 : ICoreWebView2_10 {
+  /// Add an event handler for the ServerCertificateError event.
+  /// The ServerCertificateError event is raised when the WebView2
+  /// cannot verify server's digital certificate while loading a web page.
+  ///
+  /// This event will raise for all web resources and follows the `WebResourceRequested` event.
+  ///
+  /// With this event you have several options for responding to server
+  /// certificate error requests:
+  ///
+  /// Scenario                                                   | Handled | Cancel
+  /// ---------------------------------------------------------- | ------- | ------
+  /// Ignore the warning and continue the request                | True    | False
+  /// Cancel the request                                         | n/a     | True
+  /// Display default TLS interstitial page                      | False   | False
+  ///
+  /// If you don't handle the event, WebView2 will show the default TLS interstitial page to user.
+  ///
+  /// WebView2 caches the response when `Handled` is `TRUE` for the RequestUri's host and the server certificate in the session
+  /// and the `ReceivingServerCertificateError` event won't be raised again for this host and certificate.
+  ///
+  /// To raise the event again you must clear the cache using `ClearServerCertificateErrorOverrideCache`.
+  ///
+  ///
+  /// \snippet SettingsComponent.cpp ServerCertificateError1
+  HRESULT add_ReceivingServerCertificateError(
+      [in] ICoreWebView2ReceivingServerCertificateErrorEventHandler* eventHandler,
+      [out] EventRegistrationToken * token);
+  /// Remove an event handler previously added with add_ReceivingServerCertificateError.
+  HRESULT remove_ReceivingServerCertificateError([in] EventRegistrationToken token);
+
+  /// Clears all cached decisions to proceed with TLS certificate errors from the
+  /// ReceivingServerCertificateError event for all WebView2's sharing the same session.
+  HRESULT ClearServerCertificateErrorOverrideCache(
+      [in] ICoreWebView2ClearServerCertificateErrorOverrideCacheCompletedHandler*
+      handler);
+}
+
+/// Receives the result of the `ClearServerCertificateErrorOverrideCache` method.
+[uuid(2F7B173D-3CE1-4945-BDE6-94F4C57B7209), object, pointer_default(unique)]
+interface ICoreWebView2ClearServerCertificateErrorOverrideCacheCompletedHandler : IUnknown {
+  /// Provides the result of the corresponding asynchronous method.
+  HRESULT Invoke([in] HRESULT errorCode, BOOL isSuccessful);
```

Comment:
Just the HRESULT is enough to determine if it succeeded right? Either we shouldn't have a BOOL as a second parameter here, or you need to update the C#/WinRT definition to have ClearServerCertificateError... return a bool.
```suggestion
  HRESULT Invoke([in] HRESULT errorCode);
```